src/main/java/cz/codecamp/logger/formatters/JsonFormatter.java

```diff
@@ -0,0 +1,41 @@
+package cz.codecamp.logger.formatters;
+
+import com.google.gson.Gson;
+import cz.codecamp.logger.FormatterInterface;
+import cz.codecamp.logger.LogLevelEnum;
+
+/**
+ * Created by micha on 05.10.2016.
+ */
+public class JsonFormatter implements FormatterInterface {
+
+    @Override
+    public String format( LogLevelEnum level, String message, long timeInMillis, String callingClass, int callingLineNumber ) {
+        Gson gson = new Gson();
```

Comment:
`Gson` instance by neměla být vytvářena uvnitř metody při každém volání, určitě by měla být jako field objektu. Takhle se pokaždé vytvoří a hned zahodí => zbytečný pressure na garbage collector.